style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: none;
+}
+
+/* Start of header */
+.header {
+  left: 0;
+  top: 0;
+  right: 0;
+  height: 60px;
+  width: 100vw;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+nav {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  width: 100%;
+  height: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto 14px;
+}
+
+/* End of header */
+
+/* start of main body */
+.main-body {
+  width: 412px;
+  background: url(/images/header-illsutration-mobile.png);
```

Comment:
- [ ] [ OPTIONAL ] To ensure that your images work perfectly across all browsers, kindly avoid adding a `/` at the beginning of the image url. Consider using `images/myimage.png` instead of  `/images/myimage.png`